docs/reference_behavior_attributes.md

```diff
@@ -376,6 +387,8 @@ When using `new`, the fetched content must be either a full Hyperview XML docume
 
 If the element triggering the modal is a [`<form>`](/docs/reference_form) or contained in a `<form>` element, the form's data will be serialized as query parameters and included in the request. The form data has no effect when the href refers to a local `<screen>` element.
 
+<a id="action-back"></a>
```

Comment:
Is this needed?

Reply:
It is. We now have two "back" sections. This anchor is needed to target the correct one in the links.